app/javascript/App.jsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import { Container } from 'react-bootstrap';
 import { Outlet } from 'react-router-dom';
-import CurrentUser from './components/user/CurrentUser';
+import Header from './components/shared/Header';
```

Comment:
Good point, I think changing the `/components/stylings -> /components/shared` may be more adequate.
What do you think guys?